configs/_base_/datasets/imagenet_bs128.py

```diff
@@ -0,0 +1,45 @@
+# dataset settings
+dataset_type = 'ImageNet'
+
+img_norm_cfg = dict(
+    # The mean and std are used in PyCls when training and testing RegNets
+    mean=[103.53, 116.28, 123.675],
```

Comment:
also, we may consider following our own setting when we are able to align the performance using our own models.